src/commands/install/install_package_from_repo_test.py

```diff
@@ -0,0 +1,27 @@
+from os import path
+
+import pytest
+from git.repo import Repo
+
+from src.commands.install import installation_exceptions
+from src.commands.install.install_package_from_repo import install_package_from_repo
+
+
+@pytest.fixture(name="repo_url")
+def fixture_repo_url():
+    return "https://github.com/starkware-libs/cairo-lang"
```

Comment:
I think we should't call external source from our tests, it is more problematic than it brings value.

Reply:
Agree. I want lazy here.